kvm.go

```diff
@@ -134,7 +142,7 @@ func (d *Driver) GetCreateFlags() []mcnflag.Flag {
 		mcnflag.StringFlag{
 			Name:  "kvm-io-mode",
 			Usage: "Disk IO mode: threads, native",
-			Value: "threads",
+			Value: "native",
```

Comment:
Seems to break on my system:

```
θ61° [zakame:~] % docker-machine create --driver=kvm --engine-registry-mirror=http://192.168.122.1:50000 --swarm-master swarm-manager-1
Running pre-create checks...
Creating machine...
(swarm-manager-1) Copying /home/zakame/.docker/machine/cache/boot2docker.iso to /home/zakame/.docker/machine/machines/swarm-manager-1/boot2docker.iso...
(swarm-manager-1) Creating SSH key...
(swarm-manager-1) Failed to start: virError(Code=1, Domain=10, Message='internal error: process exited while connecting to monitor: 2017-08-10T03:23:14.589561Z qemu-kvm: -drive file=/home/zakame/.docker/machine/machines/swarm-manager-1/swarm-manager-1.img,format=raw,if=none,id=drive-virtio-disk0,aio=native: aio=native was specified, but it requires cache.direct=on, which was not specified.')
Error creating machine: Error in driver during machine creation: virError(Code=1, Domain=10, Message='internal error: process exited while connecting to monitor: 2017-08-10T03:23:14.589561Z qemu-kvm: -drive file=/home/zakame/.docker/machine/machines/swarm-manager-1/swarm-manager-1.img,format=raw,if=none,id=drive-virtio-disk0,aio=native: aio=native was specified, but it requires cache.direct=on, which was not specified.')
```

Setting `kvm-cache-mode` set to `none` or `directsync`, or any other mode with explicit `cache.direct=on` got it to work again.

See also https://bugzilla.redhat.com/show_bug.cgi?id=1086704